src/app/services/notification/notification.service.ts

```diff
@@ -141,6 +144,8 @@ export class NotificationService {
 
     this.userConnection.onreconnected(() => {
       this.userConnection.invoke('Join', viewGuid, userGuid);
+      console.log('User reconnected');
+      window.location.reload();
```

Comment:
I don't think we should reload the page on reconnect. There could just be a 1 or 2 second blip in connection and we would want to reconnect and recover silently and let the user continue with whatever they were doing. It could be jarring if the page just reloads suddenly.